### PR TITLE
Added page of common proofs and realised filter of them

### PR DESCRIPTION
**Pull request by whoisalex1W**
I've created page of common proofs all the talents. Also implemented filter of them by date. 
#56

### CODE REVIEW

File: src/context/UserContext/UserContext.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const UserContext = createContext({
 });
 
 export function UserProvider({ children }) {
-    const [auth, setAuth] = useState(false);
+    const [auth, setAuth] = useState(true);
```
Comment (Ruslanchik01) [inline src/context/UserContext/UserContext.jsx]: Set to false

**Review by Ruslanchik01 — APPROVED** [review]

File: src/components/ListProofsPage/ListProofsPage.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    const filterIncrease = () => {
+        setProofs(filter(proofs));
+    };
+
+    const filterDecrease = () => {
+        const data = [...proofs].sort(
+            (a, b) => new Date(b.created) - new Date(a.created)
+        );
+        setProofs(filter(proofs).reverse());
```
Comment (daniilievtukhov) [inline src/components/ListProofsPage/ListProofsPage.jsx]: call the function filter on this method to have less lines of code ))

File: src/components/Layout/Header/Header.jsx [hunk 3]
```diff
@@ -34,7 +34,9 @@ export function Header() {
                                 return isActive ? styles.active : "";
                             }}
                         >
-                            {title}
+                            <Link to={`/${title.toLocaleLowerCase()}`}>
```
Comment (IhorKopieichykov) [inline src/components/Layout/Header/Header.jsx]: Remove child unnecessary link

File: src/components/ListProofsPage/ListProofsPage.jsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import { TalentsService } from "../../services/api-services";
+import { useState } from "react";
+import { useEffect } from "react";
+import { ProofBlock } from "../TalentPage/components/ProofBlock/ProofBlock";
+import { Button } from "../../shared/components";
+import s from "../ListProofsPage/ListProofsPage.module.scss";
```
Comment (IhorKopieichykov) [inline src/components/ListProofsPage/ListProofsPage.jsx]: Fix the import

File: src/components/ListProofsPage/ListProofsPage.jsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+export function ListProofsPage() {
+    const [proofs, setProofs] = useState({});
+
+    useEffect(() => {
+        TalentsService.getProofs().then((res) => setProofs(res));
+    }, []);
+
+    const filter = (proofs) => {
```
Comment (IhorKopieichykov) [inline src/components/ListProofsPage/ListProofsPage.jsx]:
```suggestion
    const filterByDateAsc = (proofs) => {
```

File: src/components/ListProofsPage/ListProofsPage.jsx
Comment (IhorKopieichykov) [inline src/components/ListProofsPage/ListProofsPage.jsx:1]: Make two separate methods for filtering proofs array and remove redundant set methods

**Review by IhorKopieichykov — CHANGES_REQUESTED** [review]

**Review by daniilievtukhov — APPROVED** [review]

**Review by Ruslanchik01 — APPROVED** [review]

**Review by IhorKopieichykov — APPROVED** [review]

**Pull request merged**